.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+# File introduces automated checks triggered on git events
+# to enable run `pip install pre-commit && pre-commit install`
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-docstring-first
+      - id: check-json
+      - id: check-added-large-files
+      - id: check-yaml
+      - id: debug-statements
+      - id: requirements-txt-fixer
+      - id: check-merge-conflict
+      - id: end-of-file-fixer
+      - id: sort-simple-yaml
+  - repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+      - id: black
```

Comment:
You need to add a line of config for black to set the line length in compliance with flake8 `args: ['--line-length=119']`. Example in [webref](https://code.ornl.gov/reflectometry/web_reflectivity/-/blob/next/.pre-commit-config.yaml#L30).